### PR TITLE
Josh's first year at TD post

### PR DESCRIPTION
**Pull request by JoshTGreenwood**
Dropping my first draft here for feedback / thoughts.

### CODE REVIEW

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+I can hear you groaning from across the Internet: "oh no, Josh is just sucking up to his bosses, skipping this section". Don't!
+
+What sets Todd and Justin apart is that I could fill this section with a rant about how awful they are and they would still publish this and ask me how they could be doing better. They are the very first to admit they don't have all the answers and the very last to blame anyone but themselves when things don't go well.
+
+These qualities have influenced the rest of the Team and are manifested in 2 ways.
+
+### Introspection
+Every month I have a 1 on 1 with either Todd, Justin, or both. When looking back through my notes from those meetings, I found a pattern. They ask me what *I* want to improve and then they ask me what they can do to help me meet my goals. The only thing I've been asked to do is take a close look at myself and find the areas I'm lacking. My motivation to improve has skyrocketed when I see the problems for myself and I'm not told I need to do x, y, and z.
+
```
Comment (samjonester) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: Would love to see an example here :)

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+TLDR: Todd and Justin are great and you should come work for them.
+
+I can hear you groaning from across the Internet: "oh no, Josh is just sucking up to his bosses, skipping this section". Don't!
+
+What sets Todd and Justin apart is that I could fill this section with a rant about how awful they are and they would still publish this and ask me how they could be doing better. They are the very first to admit they don't have all the answers and the very last to blame anyone but themselves when things don't go well.
+
+These qualities have influenced the rest of the Team and are manifested in 2 ways.
+
+### Introspection
+Every month I have a 1 on 1 with either Todd, Justin, or both. When looking back through my notes from those meetings, I found a pattern. They ask me what *I* want to improve and then they ask me what they can do to help me meet my goals. The only thing I've been asked to do is take a close look at myself and find the areas I'm lacking. My motivation to improve has skyrocketed when I see the problems for myself and I'm not told I need to do x, y, and z.
```
Comment (samjonester) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: I love the sentence "They ask me what I want to improve... the final formatting is a little confusing, though. I thought italicized I looked like a slash and got distracted from the power of the sentence.

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Here are a few of the things I've learned and observed.
+
+## There are more important things than writing code
+When I'm consulting for a client, I'm expected to ramp up on projects and start delivering features immediately. It's important to get a good understanding of the existing code, but I've found it even more valuable to understand the business I'm working to improve.
+
+Discovering the context around the business is a tricky problem because it usually can't be found in the README or issue tracker. Gaining that context requires talking to existing members of the team and finding out what things are going well for them, what things aren't, and what things I can be doing to deliver maximum value for the business.
+
+Before starting at Test Double, I've be happy to simply grab the top item on the issue tracker and start banging out code. The first few weeks on a team are unique because you can lend a fresh set of eyes to a business and its codebase, question assumptions, and ultimately provide more value to the business than you could heads down writing code.
```
Comment (jasonkarns) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: This paragraph seems a bit disjoint, like it's being set up for contrast with the previous paragraph, but it never comes. (Also, typo in first sentence. "I've be..")

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+I just passed my 1 year anniversary at Test Double and I'd like to share a bit about the ups and downs that came my way.
+
+A bit about me: I'm 24 years old and currently living in Louisville, Kentucky. I've been programming about 5 years. I worked as a freelance consultant and worked for a few companies in college but still had the "new college grad" badge on. I knew some stuff, but I knew that there was still so much for me to learn.
+
+Here are a few of the things I've learned and observed.
+
+## There are more important things than writing code
+When I'm consulting for a client, I'm expected to ramp up on projects and start delivering features immediately. It's important to get a good understanding of the existing code, but I've found it even more valuable to understand the business I'm working to improve.
+
+Discovering the context around the business is a tricky problem because it usually can't be found in the README or issue tracker. Gaining that context requires talking to existing members of the team and finding out what things are going well for them, what things aren't, and what things I can be doing to deliver maximum value for the business.
```
Comment (jasonkarns) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: Love this. Needs to be part of "programming 101"

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+These qualities have influenced the rest of the Team and are manifested in 2 ways.
+
+### Introspection
+Every month I have a 1 on 1 with either Todd, Justin, or both. When looking back through my notes from those meetings, I found a pattern. They ask me what *I* want to improve and then they ask me what they can do to help me meet my goals. The only thing I've been asked to do is take a close look at myself and find the areas I'm lacking. My motivation to improve has skyrocketed when I see the problems for myself and I'm not told I need to do x, y, and z.
+
+### Autonomy
+The amount of trust Todd and Justin have in their team is staggering. When I'm relieved of the stress of trying to make my bosses happy, I'm free to do my very best work and make decisions knowing that they will back me up even when I screw up. As a new grad, I expected to have to start at Test Double with the training wheels on but nothing could be further from reality. I've been treated with respect and trust at every step of the way which has made me that much more likely to be successful and deliver value at my clients.
+
+## Surrounding myself with people smarter than me makes learning a guarantee
```
Comment (jasonkarns) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: Don't sell yourself short. Knowledge (especially in tech/programming) isn't a linear spectrum. Anyone can learn something from someone else, so comparing others as "smarter than myself" does a disservice to the value and insight *you* can provide others!

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+This one should be a no-brainer for me, but it has taken some time getting used to. My instinctual reaction to being the least knowledgeable person in the room is to "act the part" and hide my own ignorance. This is counterproductive for 2 reasons: I don't grown and no one really believes me anyways. At Test Double, showing your ignorance does not result in judgment. It results in people stepping up to teach. I can't stress how valuable this has been for me.
+
+## Working From Home Can Be Hard
```
Comment (jasonkarns) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: I would love to see this section grown into its own post!

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+## In Summary
+I've grown more in 1 year at Test Double than I had in several previous years in school and consulting on my own. I still have a lot of areas for improvement, but I anticipate the same growth curve continuing in the coming year. If Test Double sounds like a place you would like to work, drop us a line at hello@testdouble.com. Test Double is always interviewing and we would love to chat.
```
Comment (jasonkarns) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: For "recruiting", `join@testdouble.com` is the best contact email.

**Comment by jasonkarns** [comment]
Great post!

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
+---
+title: "My First Year At Test Double"
+author:
+  name: "Josh Greenwood"
+  url: "http://twitter.com/searls"
```
Comment (samjonester) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: Did you mean do use @searls's twitter here?

Comment (JoshTGreenwood) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: lol no

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 title: "My First Year At Test Double"
 author:
   name: "Josh Greenwood"
-  url: "http://twitter.com/searls"
+  url: "http://twitter.com/josh"
```
Comment (samjonester) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: https://twitter.com/JoshTGreenwood ?

File: app/posts/2016-06-20-first-year-at-testdouble.md [hunk 10]
```diff
@@ -27,7 +27,7 @@ What sets Todd and Justin apart is that I could fill this section with a rant ab
...
 
 ### Introspection
-Every month I have a 1 on 1 with either Todd, Justin, or both. When looking back through my notes from those meetings, I found a pattern. They ask me what I want to improve and then they ask me what they can do to help me meet my goals. The only thing I've been asked to do is take a close look at myself and find the areas I'm lacking. My motivation to improve has skyrocketed when I see the problems for myself and I'm not told I need to do x, y, and z.
+Every month I have a 1 on 1 with either Todd, Justin, or both. When looking back through my notes from those meetings, I found a pattern. They ask me what I want to improve and then they ask me what they can do to help me meet my goals. The only thing I've been asked to do is take a close look at myself and find the areas I'm lacking. Recently, I've been focused on how I can improve communication on my client team and how we can leverage each person's skills to level each other up. On the more techinical side, I've been focused on learning clojure and iOS. My motivation to improve has skyrocketed when I see the problems for myself and I'm not told I need to do x, y, and z.
```
Comment (samjonester) [inline app/posts/2016-06-20-first-year-at-testdouble.md]: :thumbsup:

**Comment by JoshTGreenwood** [comment]
I'm :+1: with releasing this now.  Let me know if I missed anything.  If not, let's add this to the release calendar.

**Pull request merged**